### PR TITLE
Add Cypress tests step in Github action Ci

### PR DESCRIPTION
**Pull request by avdev4j**
cypress step will be moved after unit tests steps.

close https://github.com/jhipster/jhipster-control-center/issues/95

### CODE REVIEW

**Comment by avdev4j** [comment]
close until I found the right configuration (I'll test on my own repo)

**Comment by pascalgrimaud** [comment]
LGTM @avdev4j

File: .github/workflows/github-ci.yml [hunk 1]
```diff
@@ -20,22 +20,34 @@ jobs:
...
-              run: |
-                  ./mvnw -ntp --batch-mode initialize org.jacoco:jacoco-maven-plugin:prepare-agent sonar:sonar \
-                  -Dsonar.host.url=https://sonarcloud.io \
-                  -Dsonar.projectKey=jhipster_jhipster-control-center \
-                  -Dsonar.organization=jhipster \
-                  -Dsonar.login=$SONAR_TOKEN
-              env:
-                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-                SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+            #- name: Run backend test
```
Comment (pascalgrimaud) [inline .github/workflows/github-ci.yml:23]: just to know why did you comment this part @avdev4j ?

Comment (avdev4j) [reply .github/workflows/github-ci.yml:23]: It's a mistake. I did it to do quick tests. I'll fix it tomorrow.

**Pull request merged**